[0m[[0m[0mdebug[0m] [0m[0m> Exec(run 1, None, None)[0m
[0m[[0m[0mdebug[0m] [0m[0mEvaluating tasks: Compile / run[0m
[0m[[0m[0mdebug[0m] [0m[0mRunning task... Cancel: Signal, check cycles: false, forcegc: true[0m
[0m[[0m[0minfo[0m] [0m[0mCompiling 3 protobuf files to /home/james/Desktop/dev-332project/target/scala-2.13/src_managed/main/scalapb[0m
[0m[[0m[0minfo[0m] [0m[0mcompiling 17 Scala sources to /home/james/Desktop/dev-332project/target/scala-2.13/classes ...[0m
[0m[[0m[0minfo[0m] [0m[0mdone compiling[0m
[0m[[0m[31merror[0m] [0m[0mjava.io.IOException: Failed to bind to address 0.0.0.0/0.0.0.0:50051[0m
[0m[[0m[31merror[0m] [0m[0m	at io.grpc.netty.NettyServer.start(NettyServer.java:328)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.grpc.internal.ServerImpl.start(ServerImpl.java:183)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.grpc.internal.ServerImpl.start(ServerImpl.java:92)[0m
[0m[[0m[31merror[0m] [0m[0m	at network.master.Master.network$master$Master$$start(Master.scala:47)[0m
[0m[[0m[31merror[0m] [0m[0m	at network.master.Master$.main(Master.scala:26)[0m
[0m[[0m[31merror[0m] [0m[0m	at network.master.Master.main(Master.scala)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.reflect.Method.invoke(Method.java:566)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.invokeMain(Run.scala:143)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.execute$1(Run.scala:93)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.$anonfun$runWithLoader$5(Run.scala:120)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run$.executeSuccess(Run.scala:186)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.runWithLoader(Run.scala:120)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$bgRunTask$6(Defaults.scala:1981)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$termWrapper$2(Defaults.scala:1920)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.util.Try$.apply(Try.scala:213)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.internal.BackgroundThreadPool$BackgroundRunnable.run(DefaultBackgroundJobService.scala:369)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.Thread.run(Thread.java:829)[0m
[0m[[0m[31merror[0m] [0m[0mCaused by: java.net.BindException: Address already in use[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/sun.nio.ch.Net.bind0(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/sun.nio.ch.Net.bind(Net.java:459)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/sun.nio.ch.Net.bind(Net.java:448)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:134)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:562)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1334)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:973)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:260)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:356)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:503)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)[0m
[0m[[0m[31merror[0m] [0m[0m	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.Thread.run(Thread.java:829)[0m
[0m[[0m[31merror[0m] [0m[0m(Compile / [31mrun[0m) java.io.IOException: Failed to bind to address 0.0.0.0/0.0.0.0:50051[0m
[0m[[0m[31merror[0m] [0m[0mTotal time: 7 s, completed Nov 28, 2022, 7:55:32 PM[0m
[0m[[0m[0mdebug[0m] [0m[0m> Exec(idea-shell, None, None)[0m
[0m[[0m[0mdebug[0m] [0m[0m> Exec(, None, None)[0m
[0m[[0m[0mdebug[0m] [0m[0m> Exec(idea-shell, None, None)[0m
